### DISCUSSION TITLE
Multiple security settings for a SSID

### DISCUSSION THREAD

**Discussion by MaKu1974** [nzymedefense/nzyme#497] [Questions]
Hello,

how can I config the nzyme.conf  to enable more security settings for a SSID. 
I always get the alert:

SSID [XXXXYYY] was advertised with unexpected security settings [WPA1-PSK-CCMP-TKIP, WPA2-PSK-CCMP-TKIP]

It is indeed so, that I have this security settings in my wlan. I have WPA2-PSK-CCMP, WPA1-PSK-CCMP-TKIP and WPA2-PSK-CCMP-TKIP.

Thanks for your help and assistence.
Maku

**Comment by Elektronikus** [comment]
Hi MaKu1974,
I have the same issue and tried
802_11_networks: [
  {
    ssid: xxxxxxxxxx
    channels: [1,2,3,4,5,6,7,8,9,10,11,12,13,36,38,40,42,44,46,48,50,52,54,56,58,60,62,64,100,102,104,106]
    security: [
      WPA3-PSK-SAE-CCMP
      WPA2-PSK-CCMP
    ]
Which did not work. Did you try something else?

Is there a clear button for the alarm list?

**Comment by MoerkyMark** [comment]
I have the same issue
